### ISSUE TITLE
YouTube logo addition

### ISSUE THREAD

**Issue by ob-codes** [labrocadabro/communitytaught#50]
On all class page ("/class/all"), anchor tag of 'Youtube' in class card may be replaced with Youtube logo for better UX/UI.

![image](https://github.com/labrocadabro/communitytaught/assets/79637524/0f77c926-ac97-48fc-be5c-701aff9ba2ce)

**Comment by labrocadabro** [comment]
Good idea! It looks like the icon is already available, the class is .fa-youtube. if you'd like to take this one on, I'm happy to assign you.

**Comment by ob-codes** [comment]
thanks. I would love to give it a try but for now, I am unable to clone the repo. Getting an RPC failure due to big file size may be. I'll try a few things and comeback.

**Comment by ob-codes** [comment]
Update: I have successfully set up and run the project on my local machine. Would be raising a PR to add more details to readme.md 
I think font-awesome icons are not included in the project coz the icons appear like this. Will look more deeply before giving up. :)

![image](https://github.com/labrocadabro/communitytaught/assets/79637524/17fe4737-88d3-4cff-8c19-f6e2195c8e73)

**Comment by labrocadabro** [comment]
There's an [open issue](https://github.com/labrocadabro/communitytaught/issues/44) for adding a guide to getting set up locally. It's already assigned, but feel free to comment with any information you feel should be included.

It seems I made a small mistake  - you need to add both the .fab and .fa-youtube classes for the icon to show. 

For better accessibility, the link should include both the icon and the text.

**Comment by ob-codes** [comment]
thanks for the lead. I was trying classes 'fa fa-youtube' but was not getting the result. For now its looking like this.

![image](https://github.com/labrocadabro/communitytaught/assets/79637524/9b3a4949-aaca-4fed-8be4-1cc4271c7b56)

**Comment by labrocadabro** [comment]
Great, if you can just add the text back in along with the icon, you can submit the PR and I'll get it merged. :)
